models.py

```diff
@@ -612,6 +612,94 @@ class TopicModelView(RestrictedModelView):
     can_delete = True
     can_edit = False
 
+class TokenTxModelView(RestrictedModelView):
+    can_create = False
+    can_delete = False
+    can_edit = False
+
+    def _format_date(view, context, model, name):
+        if model.date:
+            return datetime.datetime.fromtimestamp(model.date).strftime('%Y-%m-%d %H:%M:%S')
+
+    def _format_json_data_html_link(view, context, model, name): 
+        ids = model.id
+        json_obj = json.loads(model.json_data)
+        assetId = json_obj["assetId"]
+        feeAssetId = json_obj["feeAssetId"]
+        senderPublicKey = json_obj["senderPublicKey"]
+        recipient = json_obj["recipient"]
+        amount = json_obj["amount"]/100
+        fee = json_obj["fee"]/100
+        timestamp = json_obj["timestamp"]
+        attachment = json_obj["attachment"]
+        signature = json_obj["signature"]
+        #txtype = json_obj["type"] ###can't use this
```

Comment:
why dont you show the type?

Reply:
added it. it was erroring before because one of the examples i had didnt have the key "type" in it.